pkg/kv/transport.go

```diff
@@ -181,9 +181,11 @@ func (gt *grpcTransport) SendNext(done chan<- BatchCall) {
 			client.args.Txn = &clonedTxn
 		}
 
-		reply, err := localServer.Batch(gt.opts.ctx, &client.args)
-		gt.setPending(client.args.Replica, false)
-		done <- BatchCall{Reply: reply, Err: err}
+		go func() {
```

Comment:
Does this have any impact on the single-node benchmarks? Just curious; I think this is the right thing to do.